lib/device_detector/client_hint.rb

```diff
@@ -177,5 +177,9 @@ def extract_model
 
       headers['Sec-CH-UA-Model']
```

Comment:
I ran into this issue too. I believe we also need to unquote `'Sec-CH-UA-Model'`, which is also wrapped, at least on MacOS Chrome.